docs/test-start-server.md

```diff
@@ -0,0 +1,38 @@
+#### test-start-server
+---
+Start a Liberty Profile server in background. The server instance will be automatically created if it does not exist. test-start-server honors the skipTests, skipITs, and maven-test-skip properties. It also allows you to bypass automatically starting the server during the pre-integration-test phase with pom configuration or a Liberty-specific command line argument.
```

Comment:
remove "Profile".  Add `tick` marks to any code-like element such as: test-start-server, skipTests, skipITs, maven-test-skip